### PR TITLE
Bug 1888551: Show Backingstores as list in the review step of creation flow of BucketClass

### PR DESCRIPTION
**Pull request by bipuladh**
Screenshot from 2020-11-09 12-31-34
/assign @shirimordechay

### CODE REVIEW

File: frontend/packages/noobaa-storage-plugin/src/components/bucket-class/wizard-pages/review-page.tsx [hunk 1]
```diff
@@ -40,14 +40,28 @@ const ReviewPage: React.FC<ReviewPageProps> = ({ state }) => {
...
+            </List>
+          </div>
         </div>
         {tier2Policy && (
           <>
             <Title size="md" headingLevel="h5" data-testid="tier2-policy">
               Tier 2: {tier2Policy}
             </Title>
-            <p data-testid="tier2-stores">Selected Backing Store: {tier2BackingStore.join(', ')}</p>
+            <div className="nb-bc-create-review__selected-stores" data-testid="tier1-stores">
```
Comment (cloudbehl) [inline frontend/packages/noobaa-storage-plugin/src/components/bucket-class/wizard-pages/review-page.tsx:57]:
```suggestion
            <div className="nb-bc-create-review__selected-stores" data-testid="tier2-stores">
```

**Comment by bipuladh** [comment]
/hold
tests are failing

**Pull request merged**